tests/performance_tests/test_import_time.py

```diff
@@ -43,21 +43,18 @@
 def test_get_ref_import_time() -> None:
     """A dummy test that would always fail, used to generate copyable reference time."""
     import_times = {
-        module_name: measure_import_time_in_ms(module_name)
+        module_name: round(measure_import_time(module_name), 2)
         for module_name in REF_IMPORT_TIME
     }
 
     # Print out the import times in a copyable format
     print("\nCopyable import time dictionary:")
-    print("{")
-    for module_name, import_time in import_times.items():
-        print(f'    "{module_name}": {import_time:.2f},')
```

Comment:
@janosh This part was intended to generate a directly copyable dict (for us to directly copy and paste to update the time recording), printing a dict directly seems to give a one-liner:

```python
Copyable import time dictionary:
{'pymatviz': 2084.25, 'pymatviz.coordination': 2342.41, 'pymatviz.cumulative': 2299.73, 'pymatviz.histogram': 2443.11, 'pymatviz.phonons': 2235.57, 'pymatviz.powerups': 2172.71, 'pymatviz.ptable': 2286.77, 'pymatviz.rainclouds': 2702.03, 'pymatviz.rdf': 2331.98, 'pymatviz.relevance': 2256.29, 'pymatviz.sankey': 2313.12, 'pymatviz.scatter': 2312.48, 'pymatviz.structure_viz': 2330.39, 'pymatviz.sunburst': 2395.04, 'pymatviz.uncertainty': 2317.87, 'pymatviz.xrd': 2242.09}
```

Reply:
i saw that. the thing is, if you paste that printed dict into a python file, `ruff` will auto-format it to multiple lines so having easier to read code seemed more important

Reply:
Fair point!